snap/snapcraft.yaml

```diff
@@ -164,9 +151,7 @@ apps:
     environment:
       VAULT_INTERVAL: "--vaultInterval=10"
       SecretService_Server: localhost
-      SecretService_CertFilePath: $SNAP_DATA/secrets/edgex-kong/server.crt
-      SecretService_KeyFilePath: $SNAP_DATA/secrets/edgex-kong/server.key
-      SecretService_TokenFolderPath: $SNAP_DATA/config/security-secrets-setup/res
+      SecretService_TokenFolderPath: $SNAP_DATA/config/security-secretstore-setup/res
```

Comment:
What is this used for, a vault token? AFAIK there are no tokens written to the directory `$SNAP_DATA/config/security-secretstore-setup/res`.

Reply:
well, that is used by toke file provider I believe. Since we are eliminating the `security-secrets-setup` and that path will be no more; so I change it to other folder path.  Note that in the original toml file it is set to `TokenFolderPath = "/vault/config/assets"`.    Should I change it to something else in $SNAP_DATA?

Reply:
After discussing this afternoon, I think the original setting was incorrect, however for some reason didn't have a negative impact. As such, I'm marking this conversation resolved.